visualization/.prettierrc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "printWidth": 140,
+    "printWidth": 120,
```

Comment:
It seems `prettier` decided to overwrite this value. I personally would continue with the 140, but that's your decision :)

Reply:
I agree with @alschmut 
Maybe this will reduce the amount of changes in this PR :)

Reply:
I looked at the changes that 140->120 brought and I'm not a fan. Sometimes method headers and other lines get broken down, even thought they're much more readable with 140 line length. At least on my laptop I can easily display the full 140 characters.

I pushed some commits that changed the printWidth to 140 again and formatted the project accordingly. 
@BridgeAR Is that fine to you?